crates/bevy_math/src/direction.rs

```diff
@@ -136,6 +136,11 @@ impl Dir2 {
     pub fn from_xy(x: f32, y: f32) -> Result<Self, InvalidDirectionError> {
         Self::new(Vec2::new(x, y))
     }
+
+    /// Returning the inner [`Vec2`]
```

Comment:
This comment format is quite strange. The verb should generally be in the present tense, not present continuous.

```suggestion
    /// Returns the inner [`Vec2`]
```

Reply:
My bad. English is not my first language.